scripts/createComponent.js

```diff
@@ -75,6 +75,30 @@ export default ${name};
   };
 };
 
+const storiesFactory = (directory, name) => {
+  return {
+    path: `${directory}/${name}.stories.tsx`,
+    template: `${copyrightHeader}
+import { Story, Meta } from '@storybook/react';
+import React from 'react';
+import { ${name}, ${name}Props } from '../../src/core';
+
+export default {
+  component: ${name},
+  argTypes: {
+    className: { control: { disable: true } },
```

Comment:
We want not to show them? I remember we discussed to maybe show them just to realize that you can use those props...?

Reply:
A wait, its control... all good